src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,8 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    /* Prefix definitions for task */
+    public static final Prefix PREFIX_TITLE = new Prefix("n/");
```

Comment:
Remember to change the prefix to what we discussed
addTask *title* -d *description*

Reply:
This is for the editTask command to be added later so that the user can specify which attribute of the task to edit